### PR TITLE
Prepare AI packages for release of 0.1.0

### PR DESCRIPTION
**Pull request by BenWilson2**
**PR Checklist**

- [X] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

Updates the package READMEs for initial release, fixes version inconsistencies, and adds a release preparation script.

### CODE REVIEW

File: ai/integrations/llama_index/README.md [hunk 1]
```diff
@@ -4,23 +4,89 @@ You can use functions defined within Unity Catalog (UC) directly as tools within
...
+pip install unitycatalog-ai[databricks]
+```
+
+### Integration Library
+
+With the appropriate client installed for the AI functionality for Unity Catalog, you can then install the `LlamaIndex` integration library:
 
 ```sh
-pip install git+https://github.com/unitycatalog/unitycatalog.git#subdirectory=ai/integrations/llama_index
+pip install unitycatalog-llamaindex
```
Comment (serena-ruan) [inline ai/integrations/llama_index/README.md:26]: Can we release unitycatalog-ai first, then update pyproject.toml file `databricks` to depend on `unitycatalog-ai[databricks]>=0.1.0`, `oss` to depend on `unitycatalog-ai[oss]>=0.1.0`

**Review by serena-ruan — APPROVED** [review]
LGTM once https://github.com/unitycatalog/unitycatalog/pull/797/files#r1881418445 is resolved!

File: ai/integrations/llama_index/README.md (hunk 1 above)
Comment (BenWilson2) [reply ai/integrations/llama_index/README.md:26]: We're actually removing the 'oss' naming convention here. I'll update this PR to reflect the changes (default will be standard Unity Catalog (from this repo), with [databricks] being an option for managed UC only).

**Review by serena-ruan — APPROVED** [review]
LGTM! https://github.com/databricks-eng/ai-oss-integration-tests-runner/actions/runs/12308411820 for validating this PR

**Pull request merged**